TinyConstraints/Classes/TinyConstraints+superview.swift

```diff
@@ -53,7 +53,7 @@
 #else
     import UIKit
     
-    public extension TinyView {
+@MainActor public extension TinyView {
```

Comment:
Not needed since `TinyView` is already isolated to the `@MainActor`

```suggestion
public extension Constraint {
```